package.json

```diff
@@ -10,7 +10,8 @@
 		"clean": "rm -rf ./dist",
 		"clean:all": "rm -rf ./dist && rm -rf ./coverage && rm -rf ./node_modules",
 		"test": "jest",
-		"test:coverage": "jest --collect-coverage"
+        "test:coverage": "jest --collect-coverage",
```

Comment:
nit: ironically this messes up formatting for the package.json 😛

Reply:
@Stvad that's because I had added package.json to prettierignore. With 4 spaces, we can just keep it as it is :)